RawWaterManagement/Spillway/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,146 @@
+{
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -26.363813,
+        -95.138688
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Treat rather scientist. Financial chair hour explain market admit.",
+      "addressLocality": "Each laugh agent tough commercial candidate great. Against second whether figure fear able. However think body paper natural bed.",
+      "addressRegion": "Local significant thing. Information teach research offer wrong mother arrive. Address effect sort run some more president.",
+      "addressCountry": "Low government tough arrive task democratic. Nature kind popular next agent.",
+      "postalCode": "Take three local bit country focus. Entire plant nice look discussion.",
+      "postOfficeBoxNumber": "Huge benefit fact many ask program. Firm condition woman writer follow billion.",
+      "areaServed": "Better now doctor agree. Together measure increase read. Want life reach red."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Ever must sort where culture room at. Person popular side simply yourself much gun dinner."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:HDWO:33110928",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1993-03-09T19:34:50Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2006-06-14T21:09:34Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "State late teach word. Throw political music."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Very red could catch bit better. Defense sell mother. Agency lead character arm drop country."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Who usually similar yeah from report few."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Less foot defense mention price work anyone. Draw young my result. Senior life long system science respond such."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Person season available. Send church food. Reduce accept enjoy lead."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:EUUN:79828978",
+      "urn:ngsi-ld:dataModel:items:REVJ:43009644"
+    ]
+  },
+  "seeAlso": {
```

Comment:
Type suggested toi be relationship
1. Relationships should not be modelled as properties, otherwise you completely lost the interest of NGSI-LD which is based on graph. 
2. Arrays in NGSI-LD are stored as strings so you can not use the API to query indiviudual alements in a array. 
3. Relationships can not be provided within arrays.

When you need to have several relationships with the same name, then you have to use the datasetId 

Example for a relationship where one entity would "monitor" 2 different rooms. It thus have 2 relationshipsnames "monitored" directed toward room01 and room02. The datasetId allows to differentiate the 2 relationships.
"monitors":[
	{
	   "type":"Relationship",
	   "object":"urn:ngsi-ld:Room:01",
	   "datasetId":"urn:ngsi-ld:dataSet:01"},
	},
	"type":"Relationship",
	"object":"urn:ngsi-ld:Room:02" ,
	"datasetId":"urn:ngsi-ld:dataSet:02"}
	]